src/Mod/Path/PathScripts/PathAreaOp.py

```diff
@@ -328,7 +327,7 @@ def _buildProfileOpenEdges(self, obj, edgeList, isHole, start, getsim):
         self.endVector = end_vector # pylint: disable=attribute-defined-outside-init
 
         simobj = None
-        if getsim and False:
+        if getsim:
             areaParams['ToolRadius'] = self.radius - self.radius * .005
             area.setParams(**areaParams)
```

Comment:
I wasn't sure what the `if getsim:` block did, other than be useful for simulations.  This containing function is a modified version of the `_buildPathArea()` method directly above.  This `if getsim:` block was left in with the `and False` restraint in case the code block was deemed necessary with additional modifications.  It looks like the block is dead, irrelevant code.  I removed it.